source/codegen/metadata/nifgen/attributes.py

```diff
@@ -1,870 +1,1354 @@
-# -*- coding: utf-8 -*-
-# This file is generated from NI-FGEN API metadata version 20.0.0d5
 attributes = {
+    1050002: {
```

Comment:
Why are the attribute changes still seen here? You might have to pull in the latest changes from the feature branch

Reply:
merged with the feature branch